packages/base/all/vendor-config-onl/src/python/onl/support/OnlSupport.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python
+from shutil import copytree, ignore_patterns, copy, make_archive, rmtree
+import logging
+import errno
+import os
+import subprocess
+import time
+
+from onl.platform.current import OnlPlatform
+
+copy_dir_list = ['/etc',
+                 '/var/log',
+                 '/mnt/onl']
```

Comment:
This looks like you are trying to copy all of /mnt/onl/* into /tmp/*. This is going to blow up.